cogment_verse/specs/spaces_serialization.py

```diff
@@ -37,9 +37,9 @@ def serialize_gym_space(gym_space, serilization_format=SerializationFormat.STRUC
         if isinstance(gym_space.n, np.ndarray):
             size = gym_space.n
         elif isinstance(gym_space.n, int):
-            size = np.array([gym_space.n], dtype=np.dtype("int8"))
```

Comment:
Serialization fix from other environment integration for MultiBinary gym spaces.